android/sdl_android/src/androidTest/java/com/smartdevicelink/test/proxy/RPCStructTests.java

```diff
@@ -183,4 +184,61 @@ public void testGetDouble() {
             fail(e.getMessage());
         }
     }
+
+    public void testClone() {
+        Hashtable<String, Object> map = new Hashtable<>();
+        String key = "test";
+        Integer value = 42;
+        map.put(key, value);
+
+        RPCStruct rpcStruct = new RPCStruct(map);
+        rpcStruct.setPayloadProtected(true);
+        RPCStruct rpcClone = rpcStruct.clone();
+
+        assertEquals(rpcStruct.getStore().get("test"), rpcClone.getStore().get("test"));
+        assertTrue(rpcClone.isPayloadProtected());
+    }
+
+    public void testHashCode() {
+        Hashtable<String, Object> map = new Hashtable<>();
+        String key = "test";
+        Integer value = 42;
+        map.put(key, value);
+
+        RPCStruct rpcStruct = new RPCStruct(map);
+
+        Hashtable<String, Object> map2 = new Hashtable<>();
+        String key2 = "test2";
+        Integer value2 = 24;
+        map2.put(key2, value2);
+
+        RPCStruct rpcStruct2 = new RPCStruct(map2);
+
+        assertEquals(rpcStruct.hashCode(), 3556536);
```

Comment:
It might be better if you created another object here and used that to compare instead of hard coding a number.